packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -217,46 +157,60 @@ Template.AssistifyCreateRequest.onCreated(function() {
 
 	instance.expertise = new ReactiveVar(''); //the value of the text field
 	instance.validExpertise = new ReactiveVar(false);
-	instance.errorMessage = new ReactiveVar(null);
+	instance.expertiseError = new ReactiveVar(null);
+	instance.titleError = new ReactiveVar(null);
 	instance.requestTitle = new ReactiveVar('');
 	instance.openingQuestion = new ReactiveVar('');
-	instance.requestTitleInUse = new ReactiveVar(undefined);
-	instance.invalidTitle = new ReactiveVar(false);
-	instance.error = new ReactiveVar(null);
-	instance.checkRequestName = _.debounce((name) => {
```

Comment:
Validation methods should have the focus on the object which is validated, not the steps of the validation themselves